Revit_oM/Elements/DraftingInstance.cs

```diff
@@ -20,19 +20,20 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
+using BH.oM.Adapters.Revit.Interface;
 using BH.oM.Adapters.Revit.Properties;
 using BH.oM.Base;
 using BH.oM.Geometry;
 
 namespace BH.oM.Adapters.Revit.Elements
 {
-    public class DraftingObject : BHoMObject
+    public class DraftingInstance : BHoMObject, IInstance 
```

Comment:
Any specific reason for this change of name? Not saying that it is wrong, but might break scripts out in production.

Reply:
Yes... I agree it may influence existing scripts however I strongly believe that currently it is really rare to use these objects directly (calling it via type name). The reason for this change is that I was trying to find commonality between Revit and BHoM and hopefully "Instance" name is more understandable than  "Object" to Revit users